GeomIO/pFIOServerBounds.F90

```diff
@@ -0,0 +1,120 @@
+#include "MAPL_Generic.h"
+module mapl3g_pFIOServerBounds
+   use mapl_ErrorHandlingMod
+   use esmf
+   use pfio
+   use gFTL2_StringVector
+   use MAPL_BaseMod
+
+   implicit none
+   private
+
+   public :: pFIOServerBounds
+
+   integer, parameter :: grid_dims = 2
+
+   type :: pFIOServerBounds
+      private
+      integer, allocatable :: local_start(:)
+      integer, allocatable :: global_start(:)
+      integer, allocatable :: global_count(:)
+      integer, allocatable :: file_shape(:)
+   contains
+      procedure :: create_server_bounds
```

Comment:
This is really a factory method.   Probably should not be a type-bound procedure.   (I've made a similar "mistake" throughout my own code.)

If we do want to leave it as a TBP, then I suggest the name "initialize" is more correct.

Reply:
changed to initialize